src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -5007,6 +5008,221 @@ private static XmlNode[] FindNodes(object obj, string propertyName, StringCompar
         }
     }
 
+    /// <summary>
+    /// Deals with DataRow objects
+    /// </summary>
+    internal class DataRowAdapter : PropertyOnlyAdapter
```

Comment:
Core adapters have always been in the same file. Some of them were moved to `ExtraAdapter.cs` previously due to missing the corresponding type in CoreCLR. Now we are gradually moving them back.